tests/testthat/test-check.R

```diff
@@ -0,0 +1,132 @@
+test_that("check_single_string_param", {
```

Comment:
praise: great to have all these tests, covering both error & nonerror behavior!  And to see date_likes and epirange_likes distinguished.